db/models.py

```diff
@@ -50,3 +61,69 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(default=datetime.datetime.now)
```

Comment:
use auto_now_add instead

Reply:
I tried but It did not work due to mocking datetime. And It would be too strange to override save methods just to pass all the tests